### PR TITLE
fix: unable to connect locked MM wallet 

### PR DESCRIPTION
**Pull request by amateima**
https://app.shortcut.com/uma-project/story/4689/unable-to-connect-my-wallet-on-umaverse

Fixed strange MM connect issue. When click `Connect`, Metamask didn't popup if locked and connected to other chain than mainnet

### CODE REVIEW

File: utils/constants.ts [hunk 1]
```diff
@@ -76,11 +79,10 @@ export function onboardBaseConfig(_chainId?: number): Initialization {
   const infuraRpc = `https://${getNetworkName(
     chainId
   )}.infura.io/v3/${infuraId}`;
-
   return {
     dappId: process.env.NEXT_PUBLIC_ONBOARD_API_KEY || "",
     hideBranding: true,
-    networkId: 1, // Default to main net. If on a different network will change with the subscription.
+    networkId: chainId, // Default to main net. If on a different network will change with the subscription.
```
Comment (amateima) [inline utils/constants.ts:85]: Initialize Onboard with the chain id of the user's wallet

**Review by Gamaranto — APPROVED** [review]

**Comment by Yuxanju** [comment]
You have to synchronize your wallet and then you debug it with bech113

File: hooks/useOnboard.tsx [hunk 2]
```diff
@@ -4,41 +4,32 @@ import Onboard from "bnc-onboard";
...
+import { ChainId } from "../utils/chainId";
 
-export function useOnboard() {
+export function useOnboard(chainId?: ChainId) {
   const { connect, disconnect, update, setError } = useConnection();
   const instance = React.useMemo(
     () =>
       Onboard({
-        ...onboardBaseConfig(),
+        ...onboardBaseConfig(chainId),
```
Comment (daywiss) [inline hooks/useOnboard.tsx:14]: i wonder if the problem here is that onboard keeps being reloaded during the lifetime of the page. we ran into this issue in across, and it created some wierd behavior

**Review by daywiss — APPROVED** [review]
seems to work from my testing

File: hooks/useOnboard.tsx (hunk 2 above)
Comment (amateima) [reply hooks/useOnboard.tsx:14]: Hmm, interesting. I didn't wasn't aware of these weird behaviour. Should I undo this change? I think Onboard will work just fine if I don't pass the chain id of the user's wallet

**Review by Tulun — APPROVED** [review]
LGTM!

**Pull request merged**